scipy/integrate/tests/test_ivp.py

```diff
@@ -296,6 +296,29 @@ def test_integration_const_jac():
         assert_allclose(res.sol(res.t), res.y, rtol=1e-14, atol=1e-14)
 
 
+def test_integration_stiff():
+    rtol = 1e-6
+    atol = 1e-6
+    y0 = [1e4, 0, 0]
+    tspan = [0, 1e8]
+
+    def fun_robertson(t, state):
+        x, y, z = state
+        return [
+            -0.04 * x + 1e4 * y * z,
+            0.04 * x - 1e4 * y * z - 3e7 * y * y,
+            3e7 * y * y,
+        ]
+
+    for method in ['Radau', 'BDF', 'LSODA']:
+        res = solve_ivp(fun_robertson, tspan, y0, rtol=rtol,
+                        atol=atol, method=method)
+
+        # If the stiff mode is not activated correctly, these numbers will be much bigger
+        assert_(res.nfev < 5000)
```

Comment:
I suppose a plain `assert` is the "modern" way here now that we use `pytest` as a runner, though perhaps not crucial.

Reply:
I changed this to use a plain `assert`. The rest of the file still uses `assert_`. I don't know the point at which everything should be switched over for pytest.